packages/veritone-react-common/src/components/FolderTree/Modals/index.js

```diff
@@ -0,0 +1,9 @@
+import DeleteFolder from './DeleteFolder';
+import ModifyFolder from './DeleteFolder';
+import CreateFolder from './CreateFolder';
+
+export {
+    DeleteFolder,
+    ModifyFolder,
+    CreateFolder
```

Comment:
Is there a reason why EditFolder is not include here?

Reply:
It's a temp file I created and forgot deleting.